### PR TITLE
boards/lorabox: Initial board port

### PR DESCRIPTION
**Pull request by MrKevinWeiss**
### Contribution description

Add board support for Lobaro LoraBox board.

### CODE REVIEW

**Comment by waehlisch** [comment]
the name of the directory name `boards/lorabox` sounds very generic. I don't know the naming policies but I would prefer `boards/lobaro-lorabox`.

**Comment by jia200x** [comment]
> the name of the directory name boards/lorabox sounds very generic. I don't know the naming policies but I would prefer boards/lobaro-lorabox.

+1

File: boards/lorabox/include/board.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+extern "C" {
+#endif
+
+/**
+ * @name    LED pin definitions and handlers
+ * @{
+ */
+#define LED0_PORT           GPIOA
+#define LED0_PIN            GPIO_PIN(PORT_A, 1)
+#define LED0_MASK           (1 << 3)
```
Comment (leandrolanzieri) [inline boards/lorabox/include/board.h]: LED pin is number 1

File: boards/lorabox/Makefile.include [hunk 2]
```diff
@@ -0,0 +1,6 @@
+## the cpu to build for
+export CPU = stm32l1
+export CPU_MODEL = stm32l151cb
+
+# add the common header files to the include path
+INCLUDES  += -I$(RIOTBOARD)/lorabox/include
```
Comment (leandrolanzieri) [inline boards/lorabox/Makefile.include]: Please include the serial terminal and the flasher setup (openocd)

File: boards/lorabox/include/periph_conf.h [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+#define CLOCK_APB1          (CLOCK_CORECLOCK / 1)
+/** @} */
+
+/**
+ * @name   Timer configuration
+ * @{
+ */
+static const timer_conf_t timer_config[] = {
+    {
+        .dev      = TIM5,
```
Comment (leandrolanzieri) [inline boards/lorabox/include/periph_conf.h]: TIM5 is not available in this MCU, as you can see in p.51 of the [datasheet](https://www.st.com/resource/en/datasheet/stm32l151cb-a.pdf). Maybe change to TIM2?

**Comment by MrKevinWeiss** [comment]
@leandrolanzieri is taking over this PR

**Comment by MrKevinWeiss** [comment]
closing in favor of #9816.

**Pull request closed without merging**